modules/sdk-core/src/bitgo/utils/mpcUtils.ts

```diff
@@ -134,7 +134,7 @@ export abstract class MpcUtils {
     if (baseCoin.getFamily() === 'eth' || baseCoin.getFamily() === 'polygon') {
       switch (params.intentType) {
         case 'payment':
-        case 'tokenTransfer':
+        case 'transferToken':
```

Comment:
@aaldemir @anshuldoshi  This commit should have had a BREAKING CHANGE comment before merge.

Reply:
the intentType is set within the SDK (not by the user) - is that still considered a breaking change?

The portion that relies on the input from the user was not changed.